src/styles/theme.js

```diff
@@ -125,6 +125,28 @@ const components = {
   navigationWidth: 300,
 };
 
+const body4Styles = {
```

Comment:
Something might have gone wrong in themes.js merging in general, fonts are off in many bodys and in balance display (see preview)

Reply:
Can you give some examples? I built it again and it looks ok..